### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by estherkim**
https://github.com/estherkim/cdn-configuration/tree/readme for preview

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,65 @@
 # cdn-configuration
 
-:construction: under construction :construction:
+## Overview
+
+https://github.com/ampproject/cdn-configuration stores the AMP versions that are being served by AMP CDNs in `configs/versions.json`. It also hosts the deployment and patching workflows for the release on-duty engineer.
+
+These workflows replace the actions on the internal release dashboard. It does not, however, replace the error monitoring systems. Please check the internal doc for links to monitor errors and performance.
```
Comment (danielrozenberg) [inline README.md:7]: I feel like this paragraph should either be removed or moved down to a "footnote for Googler" or something like that (in which case, explicitly mention that this is part of the transition process to open-source)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+
+This README contains instructions for the releases on-duty engineer. To learn more about AMP releases, please see release documation at https://github.com/ampproject/amphtml/tree/main/docs.
+
+## Deploying and promoting a release
+
+Nightly releases are cut, built and deployed automatically.
+
+Releases are promoted to channels by updating `configs/versions.json` in a PR. These PRs are created on a schedule and assiged to the on-duty engineer, to be approved and merged.
+
+Once merged, the CDNs will pick up the new versions. It should take between 30 to 90 minutes for promotions to take effect.
```
Comment (danielrozenberg) [inline README.md:17]: Cloudflare CDN is immediate (~1 minute), Google CDN is about 15-30 minutes (configs are faster than files) for existing releases and 30-90 for new releases. Do with this information what you will :)

File: README.md [hunk 3]
```diff
@@ -1,3 +1,65 @@
...
+- Saturday: none
+- Sunday: none
+
+_\*lts promotions occur every 2nd Monday of the month._
+
+For more information about schedules and channels, see https://github.com/ampproject/amphtml/blob/main/docs/release-schedule.md.
+
+## Cherry-picking a release
+
+To cherry-pick a release, run the `Cherry-pick a release` workflow. It asks for the AMP version to fix, as well as the commit shas to cherry-pick. For multiple commits, separate the shas with a space.
```
Comment (danielrozenberg) [inline README.md:35]:
```suggestion
To cherry-pick a release, run the [`Cherry-pick a release`](https://github.com/ampproject/cdn-configuration/actions/workflows/cherry-pick.yml) workflow. It asks for the AMP version to fix, as well as the commit shas to cherry-pick. For multiple commits, separate the shas with a space.
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,65 @@
...
+
+To roll back a release, find the promote PR and revert it. Do not modify `configs/versions.json` manually.
+
+Like regular promotions, it should take between 30 to 90 minutes for the roll back to take effect.
+
+## Workflow failures
+
+If any of the workflows fail, the on-duty engineer will be notified by a GitHub issue. Please resolve the failures in order to stick to the promotion schedule as close as possible.
+
+Promotion workflows can be run manually with the workflow_dispatch trigger. A successful run will result in a PR.
```
Comment (danielrozenberg) [inline README.md:61]: `workflow_dispatch` doesn't mean anything to release-on-duty peeps that never wrote a GitHub Actions yml file on their own, how about:

```suggestion
Promotion workflows can be triggered manually from the [Actions](https://github.com/ampproject/cdn-configuration/actions) tab on this repository. A successful run will result in a PR.
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,65 @@
...
+
+As a summary, promotions run on a weekly schedule:
+
+- Monday: `nightly`, `lts`\*
+- Tuesday: `nightly`, `stable`, `beta/experimental opt-in`
+- Wednesday: `nightly`, `beta/experimental traffic`
+- Thursday: `nightly`
+- Friday: `nightly`
+- Saturday: none
+- Sunday: none
```
Comment (danielrozenberg) [inline README.md:27]: Feel free to ignore, how about this formatting though?

```suggestion
| Day of the week | `nightly` | `beta`/`experimental` | `stable` | `lts` |
| --------------- | --------- | --------------------- | -------- | ----- |
| Monday          | ✔️         |                       |          | ✔️*    |
| Tueday          | ✔️         | ✔️ (`opt-in` only)     |          |       |
| Wednesday       | ✔️         | ✔️ (`traffic` only)    | ✔️        |       |
| Thursday        | ✔️         |                       |          |       |
| Friday          | ✔️         |                       |          |       |
| Saturday        |           |                       |          |       |
| Sunday          |           |                       |          |       |

File: README.md [hunk 6]
```diff
@@ -1,3 +1,65 @@
 # cdn-configuration
 
-:construction: under construction :construction:
+## Overview
+
+https://github.com/ampproject/cdn-configuration stores the AMP versions that are being served by AMP CDNs in `configs/versions.json`. It also hosts the deployment and patching workflows for the release on-duty engineer.
+
```
Comment (danielrozenberg) [inline README.md:6]:
```suggestion

The following AMP CDNs are managed by this repository:
- https://ampjs.org/ (OpenJS Foundation owned, Cloudflare hosted)
- https://cdn.ampproject.org/ (Google owned and hosted)

```

File: README.md [hunk 7]
```diff
@@ -1,3 +1,65 @@
...
+2. switch to a new branch: `git switch -c amp-release-${amp_version}-fix`  
+   (make sure that the branch starts with `amp-release-`)
+3. cherry-pick the fix: `git cherry-pick -x ${shas}`  
+   (make sure to include the `-x` flag)
+4. resolve any merge conflicts
+5. push branch to remote: `git push https://github.com/ampproject/amphtml.git`
+
+## Rolling back a release
+
+To roll back a release, find the promote PR and revert it. Do not modify `configs/versions.json` manually.
```
Comment (danielrozenberg) [inline README.md:53]: I wonder about that. There's no easy way to revert a PR from the GitHub interface (unless I missed it!). We could instruct here to trigger the relevant promote workflow with the previous version number (though that means a manual step which is potentially error prone). wdyt?

Comment (estherkim) [reply README.md:57]: It's new, there's a revert button on merged PRs.

Comment (danielrozenberg) [reply README.md:57]: Fancyyyyy!

**Comment by estherkim** [comment]
Ok! PTAL :)

**Review by danielrozenberg — APPROVED** [review]

**Pull request merged**